crates/ibc/src/core/ics03_connection/handler.rs

```diff
@@ -38,23 +34,6 @@ pub struct ConnectionResult {
     pub connection_end: ConnectionEnd,
 }
```

Comment:
The `ConnectionResult` struct and respectively the `ConnectionIdState` enum seem unneeded anymore.

Reply:
[32bbffc](https://github.com/cosmos/ibc-rs/pull/456/commits/32bbffc63e18928dccddbb43a961881204d4a3fd)